src/Mutation.tsx

```diff
@@ -47,6 +47,10 @@ export declare type MutationOptions<TData = any, TVariables = OperationVariables
   update?: MutationUpdaterFn<TData>;
 };
 
+export declare type MutationFn<TData = any, TVariables = OperationVariables> = (
+  options?: MutationOptions<TData, TVariables>,
+) => Promise<void | FetchResult<TData>>;
```

Comment:
declaring this makes it easier to use as a prop in our components. Also added `FetchResult<TData>` which was missing prior